File: code/1020/1020.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');

const n = parseInt(lines[0]);

const years = parseInt(n / 365);
const months = parseInt((n - (years * 365)) / 30);
const minutes = parseInt(n - (years * 365) - (months * 30));

console.log(`${years} ano(s)`);
console.log(`${months} mes(es)`);
console.log(`${minutes} dia(s)`);

File: code/1020/1020.cs
using System;

class URI
{
    static void Main(string[] args)
    {
        int age_in_days;

        age_in_days = int.Parse(Console.ReadLine());

        Console.WriteLine("{0} ano(s)", age_in_days / 365);
        age_in_days %= 365;
        Console.WriteLine("{0} mes(es)", age_in_days / 30);
        age_in_days %= 30;
        Console.WriteLine("{0} dia(s)", age_in_days);   
    }
}